core/src/text.rs

```diff
@@ -406,3 +425,21 @@ into_fragment!(isize);
 
 into_fragment!(f32);
 into_fragment!(f64);
+
+/// The background style of text
+#[derive(Debug, Clone, Copy)]
+pub struct Background {
```

Comment:
Would `Padding` here make sense?

Reply:
Hmm, unless we can have that horizontal padding considered by cosmic text, there'll be overlap with neighboring glyphs. There's not always whitespace on either side of the span with background.